compiler/rustc_middle/src/mir/terminator.rs

```diff
@@ -272,7 +272,8 @@ impl<'tcx> Debug for TerminatorKind<'tcx> {
 
         let unwind = match self.unwind() {
             // Not needed or included in successors
-            None | Some(UnwindAction::Continue) | Some(UnwindAction::Cleanup(_)) => None,
+            None | Some(UnwindAction::Cleanup(_)) => None,
```

Comment:
Hm, why aren't we also making `Cleanup` explicit?

Reply:
They are already printed as successors

Reply:
Cleanup is the only unwind action that includes a basic block #